uint8 quantity_of_params
uint8[] params
---
bool result
